src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -64,4 +67,36 @@ private void getAllQuotes(RoutingContext routingContext) {
       }
     });
   }
+
+  private void postNewQuote(RoutingContext routingContext) {
+    JsonObject body = routingContext.getBodyAsJson();
+    String text = body.getString("text");
+    String author = body.getString("author");
+    if (text == null) {
+      routingContext.response().setStatusCode(404)
```

Comment:
Not the right status code

Reply:
It is stated to use 404 in the README, but I guess 400 is the expected one, WDYT?

Reply:
I made a possible fix in https://github.com/tsegismont/vertx-gsoc-2018-qotd/pull/4/commits/a212f6f9a938ee9ee5650b77219a6051f54569b9.

Reply:
@BillyYccc nice catch. My bad. I will update the README